internal/nhctl/controller/service.go

```diff
@@ -268,6 +269,14 @@ func (c *Controller) getGeneratedDeploymentLabels() map[string]string {
 	}
 }
 
+func (c *Controller) sidecarContainerPrivileged(ops *model.DevStartOptions) bool {
+	if strings.Contains(ops.DevImage, "php") {
```

Comment:
Options has a variable: language.
You can use this variable to make a decision.

Reply:
I can not find this variable in 'model.DevStartOptions'

Reply:
> I can not find this variable in 'model.DevStartOptions'

https://github.com/nocalhost/nocalhost/blob/v0.6.18/internal/nhctl/profile/config_v2.go#L99
```
type DebugConfig struct {
	RemoteDebugPort int    `validate:"Port" json:"remoteDebugPort" yaml:"remoteDebugPort"`
	Language        string `validate:"Language" json:"language" yaml:"language"`
}
```

Reply:
get `language` from `*Controller.config`